File: src/cujae/inf/citi/om/factory/interfaces/IFactoryHeuristic.java
package cujae.inf.citi.om.factory.interfaces;

import cujae.inf.citi.om.generator.heuristic.Heuristic;

/* Interfaz que define como crear una objeto Heuristic*/

public interface IFactoryHeuristic {
	
	Heuristic createHeuristic(HeuristicType heuristicType);
}

File: _BHCVRP_Python_Version/BHCVRP_Python_Version/cujae_inf_citi_om/factory/interfaces/IFactoryHeuristic.py
from generator.heuristic.Heuristic import Heuristic
from factory.interfaces.HeuristicType import HeuristicType

# Interfaz que define como crear un objeto Heuristic

class IFactoryHeuristic:
    def create_heuristic(self, heuristic_type: 'HeuristicType') -> Heuristic:
        pass
